.circleci/config.yml

```diff
@@ -26,3 +32,24 @@ jobs:
       - run-bazel-rbe:
           command: bazel build //...
       - run: bazel run @graknlabs_build_tools//unused_deps -- list
+
+  test-end-to-end:
+    machine: true
+    working_directory: ~/simulation
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run-grakn-server
+      - run-bazel-rbe:
+          command: bazel run //:simulation-binary -- -k world
+      - run-bazel-rbe:
+          command: bazel test //agents/test/... --test_arg=localhost:48555 --cache_test_results=no
```

Comment:
We cannot run this with RBE, because of:
a) the Grakn Server sits locally, and RBE machine is elsewhere without access to the local machine
b) the two test steps are interdependent on each other, and RBE could possibly skip either one if it believes to have been cached

Reply:
Changing to simply use `run` for now, hoping that in the future we can run the tests upon the simulation as a single command with RBE enabled